pkg/loadbalancer/reconciler/bpf_reconciler.go

```diff
@@ -945,6 +968,42 @@ func (ops *BPFOps) updateFrontend(fe *loadbalancer.Frontend) error {
 		return fmt.Errorf("upsert service master: %w", err)
 	}
 
+	// Create wildcard service entry (port 0) for ClusterIP and LoadBalancer services
```

Comment:
This method is getting pretty big. Let's move this into a method of its own and call it from here.

This also seems wrong. We may have multiple services on the same IP but different ports. We would delete this wildcard on the first deletion on that IP which doesn't seem right. Reference counting would probably be needed.

We could also think about adding a separate BPF map for just the frontend IPs. We anyway want to go to that direction at some point so that we don't need to have all the "NodePort frontends" per service that is currently making the services map very bloated.

@brb @dylandreimerink any thoughts on this?

Reply:
I am excited to see https://github.com/cilium/cilium/issues/14118 about to be finally fixed.

Agree what @joamaki said. This resembles the NodePort IP addr problem a lot (a separate entry for the same NodePort service per node IP addr). However, I'm not sure whether it's possible to fix it in realms of this PR.

Reply:
It would be quite a big effort to move fully to a separate frontend IP map. Let's keep the scope of this PR smaller and use the original approach. We can then revisit this later.